WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmail/LoginViaEmailView.swift

```diff
@@ -38,32 +39,60 @@ public struct LoginViaEmailView: View {
     }
 
     public var body: some View {
-        VStack(spacing: 20) {
-            TextField("", text: .constant(viewModel.email))
-                .textFieldStyle(.roundedBorder)
-                .foregroundStyle(.secondary)
+        ScrollView {
```

Comment:
suggestion: increase readability by breaking this up into components and using view builders.